docs/.vuepress/config.js

```diff
@@ -4,8 +4,12 @@ const examples = require('./containers/examples');
 const sourceCodeLink = require('./containers/sourceCodeLink');
 const nginxRedirectsPlugin = require('./plugins/generate-nginx-redirects');
 const assetsVersioningPlugin = require('./plugins/assets-versioning');
+const { getLatestVersion } = require('./helpers');
 
 const buildMode = process.env.BUILD_MODE;
+const version = process.env.DOCS_VERSION || '**';
```

Comment:
Introducing helper `getCurrentVersion`is clear, and did it following your suggestion with a name:  `getBuildDocsVersion`.

However, the helper `isLatestVersion` version is not obvious anymore. Because, each time when a similar condition is checked, the data are taken from another source. Sometimes from file paths, sometimes from permalink, sometimes it uses `getLatestVersion` helper.